### ISSUE TITLE
Uncaught ReferenceError: $ is not defined

### ISSUE THREAD

**Issue by RazerZero** [concon121/atom-maven#47]

**Comment by concon121** [comment]
Hi,  May I ask what the problem is and what the solution was,  in case anyone encounters this same issue in the future? 

Thanks,. Connor

**Comment by nickbreen** [comment]
1. Open Atom
2. Error occurs
3. Error re-occurs periodically

### Environment Info

``` bash
$ which atom
/usr/bin/atom
$ which mvn
/usr/bin/mvn
$ echo $M2_HOME
/usr/share/maven
$ mvn -v
Apache Maven 3.3.9
Maven home: /usr/share/maven
Java version: 1.8.0_91, vendor: Oracle Corporation
Java home: /usr/lib/jvm/java-8-openjdk-amd64/jre
Default locale: en_NZ, platform encoding: UTF-8
OS name: "linux", version: "4.4.0-22-generic", arch: "amd64", family: "unix"
Maven installed from ubuntu repository.
```

**Atom Version**: 1.7.4
**System**: Ubuntu 16.04
**Thrown From**: [atom-maven](https://github.com/concon121/atom-maven) package, v0.13.3

### Stack Trace

Uncaught ReferenceError: $ is not defined

```
At /home/nick/.atom/packages/atom-maven/lib/ui-utils.js:53

ReferenceError: $ is not defined
    at /home/nick/.atom/packages/atom-maven/lib/ui-utils.js:53:3
    at UiUtils.clear (/home/nick/.atom/packages/atom-maven/lib/ui-utils.js:65:4)
    at UiUtils.clearText (/home/nick/.atom/packages/atom-maven/lib/ui-utils.js:82:8)
    at /home/nick/.atom/packages/atom-maven/lib/atom-maven.js:39:7
```

### Commands

```
```

### Config

``` json
{
  "core": {}
}
```

### Installed Packages

``` coffee
# User
atom-maven, v0.13.3

# Dev
No dev packages
```

**Comment by nickbreen** [comment]
The above is the "report error" info copied from Atom itself. I've not been able to resolve the issue yet. @RazerZero are you able to elaborate please?

**Comment by concon121** [comment]
I believe the easiest solution would be to simply require jquery in ui-utils, to ensure that it is always available.

I've had a quick look through the code and it seems like im very inconsistently relying on jquery being available globally, but it strikes me now that maybe it is better practice to explicitly require the dependencies you need where you need them, rather than relying on assumptions that may or may not be correct?

**Comment by concon121** [comment]
Been unable to replicate, but I've updated the code so it explicitly requires jquery where ever its needed.

Released version 0.14.0, let me know if it fixes your problem!

**Comment by nickbreen** [comment]
Updated to 0.14.0, problem is resolved.
Thanks!
